flang/test/lit.cfg.py

```diff
@@ -88,6 +88,9 @@
 # the build directory holding that tool.
 tools = [
   ToolSubst('%f18', command=FindTool('f18'),
+    extra_args=["-intrinsic-module-directory "+config.flang_intrinsic_modules_dir],
+    unresolved='fatal'),
+    ToolSubst('%bbc', command=FindTool('bbc'),
```

Comment:
Looks like something happened with the indentation.

Reply:
I think I followed the same indentation as for the f18 entry. It looks fine to me? Am I missing something?

Reply:
The first ToolSubst is indented 2 spaces less than this one on my screen.

Reply:
You're right. I'll fix this right away.